hyperspy/signal.py

```diff
@@ -39,12 +39,12 @@
 import traits.api as t
 from tlz import concat
 
-import hyperspy
+import hyperspy.io as hypio
+import hyperspy.misc.array_tools as hypat
```

Comment:
This is subjective but I am not sure if this make the code better in a sense, as in more readable.

Reply:
The file imports a dozen different modules from hyperspy independently, but then the whole of hyperspy again just for these two submodules.